docs/content/config/advanced/kubernetes.md

```diff
@@ -2,392 +2,363 @@
+This approach
 
-- Requirement to specify exposed IPs explicitly.
+- does not preserve the real client IP, so SPF check of incoming mail will fail
+- requires you to specify the exposed IPs explicitly
 
 ### Proxy port to Service
 
-The [Proxy Pod][k8s-proxy-service] helps to avoid necessity of specifying external IPs explicitly. This comes in price of complexity: you must deploy Proxy Pod on each [Node][k8s-nodes] you want to expose mailserver on.
+The [proxy pod][k8s-proxy-service] helps to avoid the necessity of specifying external IPs explicitly. This comes st the price of complexity: you must deploy a proxy pod on each [Node][k8s-nodes] you want to expose mailserver on.
 
-**Downsides**
+This approach
 
-- __Real client IP is not preserved__, so SPF check of incoming mail will fail.
+- does not preserve the real client IP, so SPF check of incoming mail will fail
 
 ### Bind to concrete Node and use host network
 
-The simplest way to preserve real client IP is to use `hostPort` and `hostNetwork: true` in the mailserver [Pod][k8s-workload-pod]. This comes in price of availability: you can talk to mailserver from outside world only via IPs of [Node][k8s-nodes] where mailserver is deployed.
+One way to preserve the real client IP is to use `hostPort` and `hostNetwork: true`. This comes in price of availability: you can talk to the mail server from outside world only via IPs of [Node][k8s-nodes] where mailserver is deployed.
 
-!!! example
+```yaml
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
 
-    ```yaml
-    kind: Deployment
-    apiVersion: extensions/v1beta1
-    metadata:
-      name: mailserver
-    # ...
-        spec:
-          hostNetwork: true
-    # ...
-          containers:
-    # ...
-              ports:
-                - name: smtp
-                  containerPort: 25
-                  hostPort: 25
-                - name: smtp-auth
-                  containerPort: 587
-                  hostPort: 587
-                - name: imap-secure
-                  containerPort: 993
-                  hostPort: 993
+metadata:
+  name: mailserver
+
+# ...
+    spec:
+      hostNetwork: true
+    
     # ...
-    ```
+      containers:
+        # ...
+          ports:
+            - name: smtp
+              containerPort: 25
+              hostPort: 25
+            - name: smtp-auth
+              containerPort: 587
+              hostPort: 587
+            - name: imap-secure
+              containerPort: 993
+              hostPort: 993
+        #  ...
+```
 
-**Downsides**
+With this approach,
 
-- Not possible to access mailserver via other cluster Nodes, only via the one mailserver deployed at.
-- Every Port within the Container is exposed on the Host side, regardless of what the `ports` section in the Configuration defines. 
+- it is not possible to access mailserver via other cluster Nodes, only via the one mailserver deployed at
+- every Port within the Container is exposed on the Host side
```

Comment:
No, you should read the sentences as

"With this approach, it is not possible to access mailserver via other cluster Nodes, only via the one mailserver deployed at"
"With this approach, every Port within the Container is exposed on the Host side"

> Is this a formatting preference to use lower case list items instead of capitalizing the first letter of a sentence? Likewise with the removal of full stops at the end.

I removed the full stops by accident.

> The common subheading/subsection **Downsides** is replaced with a generic and less consistent message "This approach" or similar, even though the focus is on clearly identifying the downsides to an approach?

Using `**` does not indicate a subheading in Markdown (Markdownlint will even report this). I think that this section clarifies the consequences of using the approach, which are not always downsides.

Reply:
> No, you should read the sentences as

Yeah, I still do when presented with a list. Personal bias of mine perhaps that the list items still are formatted like new sentences starting with capital letters. I don't mean to be too fussy about it :)

It's a little weird at the end of the doc with a single list item. Would a numbered list work better? Up to you, leave it if you think it's fine as-is.

> Using ** does not indicate a subheading in Markdown (Markdownlint will even report this).

Yes, the issue with actual heading is it's invalid for them to all have the same heading. At the time I didn't know how to resolve that, but I'm later learned of syntax to provide unique heading ids if we wanted the ToC and links to headings to work correctly.

> which are not always downsides.

Fair enough :+1: